inductiva/storage/storage.py

```diff
@@ -89,18 +91,51 @@ def _print_contents_table(contents):
     )
 
 
-def rmdir(path: str):
-    """Deletes a directory inside the user's storage in Inductiva API.
-    Currently, directories are named by the task id of the task that
-    created them, so this function can be used to delete the directory
-    of a task.
+def rmdir(path: str, /, confirm: bool = False):
+    """Deletes the paths inside the user's storage in Inductiva API.
+
+    This function can be used to remove a single path within the user's storage,
+    for example, if `path=task_id` one can remove the path of a specific task; 
+    or, users can remove all of the contents in their storage by setting
+    `path="/"`. Note, this doesn't allow to remove specific files within 
+    any directory.
+
+    E.g.:
+        Works:
+        >>> inductiva.storage.rmdir("task_id", confirm=True)
+        >>> inductiva.storage.rmdir("/", confirm=True)
+
+        Fails:
+        >>> inductiva.storage.rmdir("task_id")
```

Comment:
```suggestion
    This function removes the given `path` from the user's remote storage
    in the Inductiva platform. The given path can either be the id of a task to
    remove all content pertaining to that task, or `*` to remove all content.
    The function requires an explicit `confirm=True` keyword argument
    to confirm that the path is, indeed, to be removed.

    E.g.:
        #Works - remote path "task_id" exists and user is explicitly
        # sets confirm=True:
        >>> inductiva.storage.rmdir("task_id", confirm=True)
        
        #Works - The user explicitly sets confirm=True, confirming that
        # all contents in his remote storage space are to be removed:
        >>> inductiva.storage.rmdir("/", confirm=True)

        # Fails -  remote path "task_id" exists but the user fails to
        # confirm his intent. The call will fail win a runtime exception:
        >>> inductiva.storage.rmdir("task_id")
```

Reply:
Here, it is not necessarily the id of the task, since user's have flexibility to select the name of where a simulation is saved in storage.